Makefile

```diff
@@ -125,6 +125,11 @@ tidy:
 	@echo "SOME TESTS WILL FAIL IF NOT RUN AS ROOT!"
 	$(GO) test github.com/vmware-tanzu/antrea/test/integration/...
 
+
+.PHONY: install-cloud-tool
+install-cloud-tool:
```

Comment:
I'd rather not have this target TBH. What purpose is it serving? There is no reason for `${HOME}/terraform` to be in the path. Since these scripts are likely to be for developers only, I'd rather have them use the `hack/aks` path.

Reply:
sound fine. will do